README.md

```diff
@@ -173,6 +174,35 @@ cppimport.settings['lock_timeout'] = 10*60 # 10 mins
 
 You should not use `force_rebuild` when importing concurrently.
 
+### Acquiring the lock hangs or times out unexpectedly - what's going on?
+Certain filesystems do not support file locking. You can disable the lock
```

Comment:
Mind giving an example or two so people might have a better guess of whether their OS supports locking or not?